bin/util

```diff
@@ -53,3 +53,55 @@ function get_conjur_admin_api_key(){
   admin_api_key=$(docker-compose exec -T conjur conjurctl role retrieve-key dev:user:admin | tr -d '\r')
   echo $admin_api_key
 }
+
+function conjur_alive(){
+  if [ -z `docker-compose ps -q conjur` ]; then
+    echo 1
+  elif [ -z `docker ps -q --no-trunc | grep $(docker-compose ps -q conjur)` ]; then
+    echo 1
+  else
+    echo 0
+  fi
+}
+
+function enterprise_alive(){
+  if [ -d "test/dap-intro" ]; then
+    pushd test/dap-intro > /dev/null
+    if [ -z `docker-compose ps -q conjur-master-1.mycompany.local` ]; then
+      echo 1
+    elif [ -z `docker ps -q --no-trunc | grep $(docker-compose ps -q conjur-master-1.mycompany.local)` ]; then
+      echo 1
+    else
+      echo 0
+    fi
+    popd > /dev/null
+  else
+    echo 1
+  fi
+}
+
+function swagger_alive(){
+  if [ -z `docker container ls -q --filter name=swagger-editor` ]; then
+    echo 1
+  else
+    echo 0
+  fi
+}
+
+function docker_network_alive(){
```

Comment:
I don't see the need for this function. It is only used to check if the network is alive before killing it, so you could just use the same trick of adding a `|| true` after the command to make sure it always exits okay even if the network isn't up.